### PR TITLE
Add kube-worker security group. Release version 1.93

### PR DESCRIPTION
**Pull request by alexeyklyukin**

### CODE REVIEW

File: acid/senza/templates/base.py [hunk 1]
```diff
@@ -417,6 +429,7 @@ def set_default_variables(variables):
     variables.setdefault('environment', 'live')
     variables.setdefault('spilo_postgres_version', SPILO_POSTGRES_VERSION)
     variables.setdefault('overcommit_memory', 2)
+    variables.setdefault('kube_worker_sg_id', 'sg-5012f03b')
```
Comment (a1exsh) [inline acid/senza/templates/base.py]: Well, no. :-p

Comment (a1exsh) [inline acid/senza/templates/base.py]: Cannot we at least do the same trick we did with odd and zmon SGs?

**Comment by a1exsh** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**